aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -139,23 +206,20 @@ module aptos_framework::genesis {
     fun create_initialize_validators(aptos_framework: &signer, validators: vector<ValidatorConfiguration>) {
         let i = 0;
         let num_validators = vector::length(&validators);
+        let unique_accounts = vector::empty();
+
         while (i < num_validators) {
             let validator = vector::borrow(&validators, i);
-            let owner = &account::create_account(validator.owner_address);
-            let operator = owner;
-            // Create the operator account if it's different from owner.
-            if (validator.operator_address != validator.owner_address) {
-                operator = &account::create_account(validator.operator_address);
-            };
-            // Create the voter account if it's different from owner and operator.
-            if (validator.voter_address != validator.owner_address &&
-                validator.voter_address != validator.operator_address) {
-                account::create_account(validator.voter_address);
-            };
-
-            // Mint the initial staking amount to the validator.
-            coin::register<AptosCoin>(owner);
-            aptos_coin::mint(aptos_framework, validator.owner_address, validator.stake_amount);
+
+            assert!(
+                !vector::contains(&unique_accounts, &validator.owner_address),
```

Comment:
Why do we need this unique_accounts?

Reply:
And why is owner added there but not operator and voter?

Reply:
why do we care if operator or voter are unique?
an owner can only stake once... so it would probably fail anyway, so we could remove it anyway

Reply:
So this is just to ensure the validator owners are unique?

Reply:
Yes